src/components/textToSpeech/js/TextToSpeech.js

```diff
@@ -142,6 +142,14 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         }
     });
 
+    // Issue functions to the speechSynthesis interface after a delay; this
+    // makes the wrapper behave better when issuing commands in rapid succession
+    fluid.textToSpeech.asyncSpeechSynthesisControl = function (control, delay) {
```

Comment:
I *really* wanted to approve this and had written a comment declaring that it was probably "good enough" but I think on balance that it probably isn't. There is no guarantee that timeouts scheduled at closely spaced instants in time will be honoured in the order that they are issued - here is some art: http://stackoverflow.com/questions/1776239/are-equal-timeouts-executed-in-order-in-javascript
I don't think there's an alternative to explicitly managing a queue of activities in the component. It will also make the body of this method a little more slightly.